scripts/githooks/commit_msg.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+
+# Check a Git commit message according to the seven rules of a good commit message:
+# https://chris.beams.io/posts/git-commit/
+import sys
+
+class GitCommitMessage:
+    'Represents a parsed Git commit message'
+
+    rules = [ 'Separate subject from body with a blank line',
```

Comment:
Nit: Seems it would be nicer to use `dict` here and map from rule name -> func (thus avoiding two unrelated arrays)